index.html

```diff
@@ -15,15 +15,148 @@
 
     <!-- We are using a CSS Reset file to standardize browser default css -->
     <link rel="stylesheet" href="reset.css" />
-
-    <!-- This is where all of our awesome CSS code goes -->
     <link rel="stylesheet" href="style.css" />
+    <link href="Mina/Mina-Regular.ttf" rel="stylesheet"> <!-- this is the logo-->
   </head>
-  <body>
-    <!--
-
-      YOU HAVE TO MODIFY THE HTML TO BUILD YOUR AWESOME PAGE
+  <main>
+  <header>
+        <h1>MeLisa</h1>  <!--using the font "mina" from google for the logo-->
+        <h2>Lisa Jonas</h2> 
+        <h3 class="header-subtitle">PHOTOGRAPHER & DESIGNER</h3>
+        <nav>
 
-    -->
-  </body>
+          <span>Home</span>
+          <img src="dot.png" alt="dot">
+          <span>Store</span>
+          <img src="dot.png" alt="dot">
+          <span>Blog</span>
+          <img src="dot.png" alt="dot">
+          <span>Resume</span>
+          <img src="dot.png" alt="dot">
+          <span>Contact</span>
```

Comment:
These should be marked up as `<a href="#">Home</a>` since they are links that would take a user to a different page.

Reply:
You could do something like 

```html
<ul class="my-navigation">
  <li><a href="#>Home</a></li>
  <li><a href="#>Store</a></li>
  <li><a href="#>Blog</a></li>
  ...
</ul>
```